nilearn/input_data/multi_nifti_masker.py

```diff
@@ -211,6 +214,73 @@ def fit(self, imgs=None, y=None):
         self.mask_img_.get_data()
         return self
 
+    def transform_imgs(self, imgs_list, confounds=None, copy=True, n_jobs=1):
+        """Prepare multi subject data in parallel
+
+        Parameters
+        ----------
+
+        imgs_list: list of Niimg-like objects
+            See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
+            List of imgs file to prepare. One item per subject.
+
+        confounds: list of confounds, optional
+            List of confounds (2D arrays or filenames pointing to CSV
+            files). Must be of same length than imgs_list.
+
+        copy: boolean, optional
+            If True, guarantees that output array has no memory in common with
+            input array.
+
+        n_jobs: integer, optional
+            The number of cpus to use to do the computation. -1 means
+            'all cpus'.
+        
+        Returns
+        -------
+        region_signals: list of 2D numpy.ndarray
+            List of signal for each element per subject.
+            shape: list of (number of scans, number of elements)
+        """
+
+        if not hasattr(self, 'mask_img_'):
+            raise ValueError('It seems that %s has not been fitted. '
+                             'You must call fit() before calling transform().'
+                             % self.__class__.__name__)
+        target_fov = None
+        if self.target_affine is None:
+            # Force resampling on first image
+            target_fov = 'first'
+
+        niimg_iter = _iter_check_niimg(imgs_list, ensure_ndim=None,
+                                       atleast_4d=False,
+                                       target_fov=target_fov,
+                                       memory=self.memory,
+                                       memory_level=self.memory_level,
+                                       verbose=self.verbose)
+
+        if confounds is None:
+            confounds = itertools.repeat(None, len(imgs_list))
+
+        # Ignore the mask-computing params: they are not useful and will
+        # just invalidate the cache for no good reason
+        # target_shape and target_affine are conveyed implicitly in mask_img
+        params = get_params(self.__class__, self,
+                            ignore=['mask_img', 'mask_args', 'mask_strategy',
+                                    'copy'])
+
+        func = self._cache(filter_and_mask,
+                          ignore=['verbose', 'memory', 'memory_level', 'copy'])
+        data = Parallel(n_jobs=n_jobs)(
+            delayed(func)(imgs, self.mask_img_, params,
+                           memory_level=self.memory_level,
+                           memory=self.memory,
+                           verbose=self.verbose,
+                           confounds=cfs,
+                           copy=copy)
+            for imgs, cfs in izip(niimg_iter, confounds))
+        return list(zip(*data))[0]
```

Comment:
My suggestions:
`return list(map(lambda x: x[0], data))`
or
`return [d[0] for d in data]]`